PhoneBook.Doc415/Enums.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace PhoneBook.Doc415;
+
+internal class Enums
+{
+    public enum MainMenuSelections
+    {
+        AddContact,
```

Comment:
it's good practice to create first enum as `None` or `Undefined` because if you create Enum without any initialization of it. It will have now `AddContact` default value, which might cause logic errors while app is running

Reply:
Thanks for the warning. I have to review enums in c#